eservice/bin/es-stop.sh

```diff
@@ -14,29 +14,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-F_USAGE='-b|--base port -c|--count services'
-F_BASE=7101
+F_SERVICEHOME="$( cd -P "$( dirname ${BASH_SOURCE[0]} )/.." && pwd )"
+source ${F_SERVICEHOME}/bin/common.sh
+
+F_USAGE='-c|--count services -b|--base name'
 F_COUNT=1
+F_BASENAME='eservice'
 
 # -----------------------------------------------------------------
 # Process command line arguments
 # -----------------------------------------------------------------
 TEMP=`getopt -o b:c:h --long base:,count:,help \
-     -n 'stop.sh' -- "$@"`
+     -n 'es-stop.sh' -- "$@"`
 
 if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
 
 eval set -- "$TEMP"
 while true ; do
     case "$1" in
-        -b|--base) F_BASE="$2" ; shift 2 ;;
+        -b|--base) F_BASENAME="$2" ; shift 2 ;;
         -c|--count) F_COUNT="$2" ; shift 2 ;;
         --help) echo $F_USAGE ; exit 1 ;;
 	--) shift ; break ;;
 	*) echo "Internal error!" ; exit 1 ;;
     esac
 done
 
-for port in `seq $F_BASE $(($F_BASE+$F_COUNT))` ; do
-    curl -s "http://localhost:$port/shutdown"
+for index in `seq 1 $F_COUNT` ; do
+    IDENTITY="${F_BASENAME}$index"
+    echo stopping enclave service $IDENTITY
+
+    url="$(get_url_base $IDENTITY)/shutdown" || { echo "no url found for enclave service"; exit 1; }
+    resp=$(curl -sL -w "%{http_code}\\n" ${url} -o /dev/null)
+    if [ $? != 0 ]; then # shutdown results in 500 error, so do not test also || [ $resp != "200" ]
+	echo "enclave service $IDENTITY not running or not properly shut down"
```

Comment:
if shutdown of one service fails, isnt hutdown attempted for the remaining ones?

Reply:
i guess you are right, will change to continue (but still make sure there is a non-zero exit code ...

> if shutdown of one service fails, isnt hutdown attempted for the remaining ones?
i guess you are right, will change to continue (but still make sure there is a non-zero exit code ...

Reply:
Corresponding change is pushed now to PR ...